exp_build/plugin.js

```diff
@@ -31,20 +31,55 @@ function setCurr(component, targetPath, subTaskFlag = false) {
   return { ...obj, isCurrentItem: isCurrentItem };
 }
 
+
+function isURL(source) {
```

Comment:
This is duplicated, I think it is present in another file as well. Please check

Reply:
Yeah it was duplicated I fixed it